tgbot/filters/admin.py

```diff
@@ -0,0 +1,18 @@
+import typing
+
+from aiogram.dispatcher.filters import BoundFilter
+
+from tgbot.config import Config
+
+
+class AdminFilter(BoundFilter):
+    key = 'is_admin'
+
+    def __init__(self, is_admin: typing.Optional[bool] = None):
+        self.is_admin = is_admin
+
+    async def check(self, obj):
+        if self.is_admin is None:
+            return True
+        config: Config = obj.bot.get('config')
+        return str(obj.from_user.id) in config.tg_bot.admin_ids
```

Comment:
inversed logic. 
admin_ids should be all integers because user ids are always integers.
We can do convertion during config reading, instead of converting to string on each asccess